### ISSUE TITLE
Docs: Show v9.0.0-alpha.0 in dropdown instead of NEXT

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#17943]
### Docs page(s)

https://eslint.orgs/docs/next

### What documentation issue do you want to solve?

Currently, the dropdown for docs shows three options: v8.56.0, HEAD, and NEXT. It's unclear what NEXT refers to.

### What do you think is the correct solution?

Instead of NEXT, display v9.0.0-alpha.0 (and any future prerelease versions) to make it clear what version these docs refer to.

### Participation

- [ ] I am willing to submit a pull request for this change.

**Comment by mdjermanovic** [comment]
> Instead of NEXT, display v9.0.0-alpha.0 (and any future prerelease versions) to make it clear what version these docs refer to.

I agree that it would be better to display the current prerelease version instead of NEXT.

This can be easily done on `main` (HEAD) and `next` branches, as they both have the current prerelease version in package.json. But the `latest` branch has no information about what the current prerelease version is, so it looks like it's time for a long-term solution that would also cover the branch for v8.56.0 we'll have when we release the final v9.0.0.

As suggested in https://github.com/eslint/eslint/issues/17893#issuecomment-1866717652, we could maintain the list in a file in the `main` branch and pull it from there.

I was thinking about fetching the list in the browser, but that wouldn't work for users who don't have JavaScript enabled.

So it looks like the solution is to fetch the list in build time, and after each release trigger rebuilds on all branches that are not already updated during the release?

**Comment by nzakas** [comment]
> So it looks like the solution is to fetch the list in build time, and after each release trigger rebuilds on all branches that are not already updated during the release?

That's what I was thinking, yes. I wonder if it's possible to leverage the stats we gather for the eslint.org homepage for this purpose?

**Comment by nzakas** [comment]
Working on this.
